src/csharp/GRPC-TOOLS-MSBUILD.md

```diff
@@ -0,0 +1,215 @@
+# Grpc.Tools MSBuild integration overview
+
+This is an overview for maintainers of Grpc.Tools.
+
+The Grpc.Tools NuGet package provides custom build targets to make it easier to specify ._proto_ files
+in a project and for those files to be compiled and their generated files to be included in the project.
+
+# Files in the NuGet package
+
+## .props and .target files
+
+MSBuild properties and targets included from the Grpc.Tools NuGet package are in:
+
+* _build\Grpc.Tools.props_, which imports
+    * _build\\\_grpc\\\_Grpc.Tools.props_
+    * _build\\\_protobuf\\Google.Protobuf.Tools.props_
+* _build\Grpc.Tools.targets_, which imports
+    * _build\\\_grpc\\\_Grpc.Tools.targets_
+    * _build\\\_protobuf\\Google.Protobuf.Tools.targets_
+
+Details of how NuGet packages can add custom build targets and properties to a project is documented
+here: [MSBuild .props and .targets in a package](https://learn.microsoft.com/en-us/nuget/concepts/msbuild-props-and-targets)
+
+Basically the ._props_ and ._target_ files are automatically included in the projects - the ._props_ at the top
+of the project and the ._targets_ are added to the bottom of the project.
+
+## Visual Studio property pages
+
+For Visual Studio integration - these files provide the properties pages:
+
+* _build\_protobuf\Protobuf.CSharp.xml_ (included from _Google.Protobuf.Tools.targets_)
+* _build\_grpc\Grpc.CSharp.xml_ (included from __Grpc.Tools.targets_)
```

Comment:
gets, rendered as "included from _Grpc.Tools.targets" so probably a mistake? (or there's a mistake in the line above)